sdk/src/types/block/output/delegation.rs

```diff
@@ -323,9 +323,7 @@ impl DelegationOutput {
         unlock: &Unlock,
         context: &mut SemanticValidationContext<'_>,
     ) -> Result<(), TransactionFailureReason> {
-        self.unlock_conditions()
-            .locked_address(self.address(), context.transaction.creation_slot())
-            .unlock(unlock, context)
+        self.address().unlock(unlock, context)
```

Comment:
I know the other version is even more verbose now but I wonder if we shouldn't keep it. Just thinking it's a bit error prone if ever we change the conditions.

Reply:
It's also some useless checks, but not a big issue, reverted